docs/topics/mta-web-proxy-config.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-proxy-config_{context}"]
+= Configuring HTTP and HTTPS proxy settings
+
+This management module configures HTTP & HTTPS proxy settings.
+
+.Procedure
+
+. Click the radio button to enter the following information.
+. In the left menu of the *Administrator* view, click *Proxy*.
+. Click *HTTP proxy* or *HTTPS proxy* toggle switches to enable the proxy connection.
+. Enter the following information:
+* Proxy host
+* Proxy port
+. Click *HTTP proxy credentials* or *HTTPS proxy credentials* toggle switches to enable authentication (Optional).
+. Select the credential from the drop-down list.
```

Comment:
IBM Style Guide says to avoid "drop-down" unless there is more than one type of list. Older documentation used to specify what type of entity was involved "Click the *Insert* button," "Select the item from the drop-dpwn list." Now, with users so used to UIs, we can simply write "Click *Insert*." or "Select from the item from the list."

Reply:
Click *Insert*.